devito/visitors.py

```diff
@@ -143,6 +149,51 @@ def always_rebuild(self, o, *args, **kwargs):
         return o._rebuild(*new_ops, **okwargs)
 
 
+class PrintAST(Visitor):
```

Comment:
Can we add a default visit_Node that at least does proper indentation and uses the class name as return value ?